src/core/MOM_barotropic.F90

```diff
@@ -4622,7 +4622,7 @@ subroutine barotropic_init(u, v, h, eta, Time, G, GV, US, param_file, diag, CS,
                  "Setting DTBT to 0 is the same as setting it to -0.98. "//&
                  "The value of DTBT that will actually be used is an "//&
                  "integer fraction of DT, rounding down.", units="s or nondim",&
-                 default = -0.98)
+                 default = -0.98, scale=1.0)
```

Comment:
How does `scale=1.0` work when DTBT>0 and is seconds? Dimensional scaling might mean we have to stop giving meaning to special values...

Reply:
The parameter DTBT is a special case, in that it is in seconds if it is positive, and it is a nondimensional fraction of the largest stable timestep when it is negative. There is currently no way to rescale a variable conditionally on its sign, so the variable is read in to a temporary variable (dtbt_in) without rescaling, and the actual variable (CS%dtbt or CS%dtbt_fraction) is rescaled or not depending on which one is being used.

  The `scale=1.0` argument is used if an input variable is intended to be in MKS units or is already nondimensional. I plan to use this to emphasize those variables where someone has actually thought about what the scaling factor should be, rather than just using a lazy default, but it has no impact on the solutions themselves, as 1.0 is the effective default value of this optional argument.

Reply:
This commit has been amended to remove this scale=1.0 argument.